_posts/randy/2022-06-24-reflecting-on-the-future.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Reflecting on the Future - A Mirror Sample of Thinks to Come"
+permalink: /@randy/reflecting-on-the-future
+ext_image: "assets/img/sample-in-progress.png"
+description: I am Old School human; equipped with analog gears but upgraded with digital belts. Having a fair amount of past, I spend a lot of time reflecting on the past. It seems like I always have a need to measure and gauge my present with the paths and events it took to get me here.
```

Comment:
![image](https://user-images.githubusercontent.com/1812989/175571208-1823c449-04d3-4f9d-abbf-777c072db5b7.png)

This description shows up like this when sharing.  I think trimming down to a single sentence with a brief description would be good to do

Reply:
Card/desc looks good - does the latter appear in search?